index.html

```diff
@@ -4,242 +4,245 @@
-                          <li class="cs">css</li>
-                          <li class="jvs">javascript</li>
-                      </ul>
-                    </div>
-                    <div class="buttn">
-                        <button type="button">See Project</button>
-                    </div>
-                  </div>
-                  <div class="card4">
-                    <a href="#"><img src="images/card4.png" alt="placeholder" class="placeholder"></a>
-                    <div class="cardtext">
-                        <h4 class="stories">Multi-Post <br> Stories</h4>
-                        <ul class="client">
-                            <li class="canopy">CANOPY</li>
-                            <li><img src="images/Counter.png" alt="counter"></li>
-                            <li class="back">Back End Dev</li>
-                            <li><img src="images/Counter.png" alt="counter"></li>
-                            <li class="yearr">2015</li>
-                        </ul>
-                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    </div>
-                    <div class="langy">
-                      <ul class="lang">
-                          <li class="htm">html</li>
-                          <li class="cs">css</li>
-                          <li class="jvs">javascript</li>
-                      </ul>
-                    </div>
-                    <div class="buttn">
-                        <button type="button">See Project</button>
-                    </div>
-                  </div>
+
+    <section class="mywork-section">
+        <div class="mydet">
+        <div class="card1 cards">
+        <div class="placeholder">
+            <a href="#"><img src="images/card1.jpg" class="placeholder mobi" alt="placeholder"></a>
+           <!-- <a href="#"><img src="images/card1.svg" class="placeholder desktp" alt="placeholder"></a>-->
+        </div>
+        <div class="descrpt">
+            <h3>Tonic</h3>
+            <ul class="client">
+                <li class="canopy cli"><a href="#">CANOPY</a></li>
+                <li class="counter"><img src="images/Counter.jpg" alt=""></li>
+                <li class="back cli"><a href="#">Back End Dev</a></li>
+                <li class="counter"><img src="images/Counter.jpg" alt=""></li>
+                <li class="dat cli"><a href="#">2015</a></li>   
+             </ul>
+            <p>A daily selection
+                 of privately personalized reads; no accounts or sign-ups required.</p>
+             <ul class="langgy">
+                <li class="html lan"><a href="#">Html</a></li>
+                <li class="css lan"><a href="#">CSS</a></li>
+                <li class="java lan"><a href="#">Javascript</a></li>
+             </ul>  
+        <div class="button">
+            <button type="button" class="buttn">See Project</button>
+        </div>
+        </div>
```

Comment:
- [ ] Technologies used in each project card along with the `See Project` button are not quite similar to the Figma design, Kindly stick to the design styles since it is a project requirement.
- [ ] Same styling issues on the mobile version, please make sure to fix them as well.

![image](https://user-images.githubusercontent.com/33767951/139549559-f2ab8a22-8962-4f0a-a092-d11283155fbd.png)

Reply:
The font color issue is taking me quite some time to fix. With the review deadline approaching, I am hoping for this option to be kindly considered optional. I will deal with it before the beginning of the next block

Reply:
It displays the right color in my desktop but a different deeper one in the github pages